docs/applications/containers/troubleshooting-kubernetes/index.md

```diff
@@ -120,21 +127,29 @@ Enter `exit` to later leave this shell.
 
 ### Viewing Master and Worker Logs
 
-The locations of logs on your master nodes are:
+If you have set up Kubernetes using a method that does not include [**systemd**](https://linode.com/docs/quick-answers/linux-essentials/what-is-systemd/), the location of logs on your master nodes are in most cases as follows:
 
 - `/var/log/kube-apiserver.log` - [API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
 - `/var/log/kube-scheduler.log` - [Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/ )
 - `/var/log/kube-controller-manager.log` - [Replication controller manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/)
 
-    {{< note >}}
-You will not be able to directly access your master nodes if you are using Linode's LKE service.
-{{< /note >}}
-
 On your worker nodes:
 
 - `/var/log/kubelet.log` - [Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
 - `/var/log/kube-proxy.log` - [Kube proxy](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/)
 
+If you do have systemd implemented, then the above logs will not be accessible and you'll need to access kubelet logs using `journalctl` instead. Below is an example command for accessing kubelet logs through journalctl:
```

Comment:
Rewording this a bit: "If your nodes run systemd", and dropping the bit about the logs not being accessible, because it'll be implied from the H4 section titles